packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -179,27 +179,25 @@ export async function setSourceBundleUrl(addinId: string, components: SourceBund
   }
 }
 
-export async function setWebView(addinId: string, webview: WebViewType | undefined): Promise<void> {
+export async function setWebView(addinId: string, webView: WebViewType | undefined): Promise<void> {
```

Comment:
I would use `webViewType` as the variable name. It starts lowercase to distinguish from the enum WebViewType.